app/services/albums.js

```diff
@@ -0,0 +1,15 @@
+const request = require('request-promise');
+
+exports.getAlbumsApi = () =>
+  request('https://jsonplaceholder.typicode.com/albums')
+    .then(htmlString => JSON.parse(htmlString))
+    .catch(error => {
+      throw error;
+    });
+
+exports.getAlbumsPhotosApi = idAlbum =>
```

Comment:
the same comments from behind, and think about making a more generic method

Reply:
Perfect!